docs/tutorials/configure/scuttlebot.md

```diff
@@ -16,138 +16,219 @@ languages: [ "python", "go" ]
+{{< tabs name="Configure an pi Board" >}}
+{{% tab name="Config Builder" %}}
 
-## Configuring the motors
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter `local` as the name for your board, select the type `board`, and select the `pi` model.
 
-### Adding the right motor
+Click **Create component**.
 
-The next step is to add a motor and make it spin a wheel.
+![The component configuration panel for a board component.](/tutorials/scuttlebot/board-empty-json.png)
 
-1. Begin by adding the right motor, naming the component "right".
-2. Select "motor" from the **Type** drop-down.
-3. Select "gpio" from the **Model** drop-down.
-4. Click **Create Component**, which will generate the motor component panel.
-5. Then select `local` from the **Board** drop-down (since the motor is wired to the Raspberry Pi named "local").
-6. Set **Max RPM** to `100`.
-7. Next, you'll need to describe how the motor is wired to the Pi:
-   1. Switch the Component Pin Assignment Type to **In1/In2**.
-   2. Set **A/In1** to `16`.
-   3. Set **B/In2** to `15`.
-   4. Leave the `pwm` (pulse-width modulation) pin blank, because this specific motor driver’s configuration does not require a separate PWM pin.
+Click on **Show more** and add `I2C` with **name** `main` and **bus** `1`.
 
-![Screenshot of the motor config panel with the attributes set as described above.](/tutorials/scuttlebot/pi-rhwheel.png)
-<br><br>
+{{% /tab %}}
+{{% tab name="Raw JSON" %}}
 
-{{% expand "Click to view the raw JSON for the right motor" %}}
+Add the following JSON object to the `components` array:
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
-  "name": "right",
-  "type": "motor",
-  "model": "gpio",
+  "type": "board",
+  "model": "pi",
+  "name": "local",
   "attributes": {
-    "pins": {
-      "a": "16",
-      "b": "15",
-      "pwm": ""
-    },
-    "board": "local",
-    "max_rpm": 100,
-    "dir_flip": false
-  },
-  "depends_on": []
+    "i2cs": [
+      {
+        "bus": "1",
+        "name": "main"
+      }
+    ]
+  }
 }
 ```
 
-{{% /expand %}}
+{{% /tab %}}
+{{< /tabs >}}
 
-{{% alert title="Tip" color="tip" %}}
-If your wheel turns in reverse when it should turn forward, add the `dir_flip` attribute (found by clicking **Show more**) and set it to "true."
-{{% /alert %}}
+## Configure the encoders
 
-### Testing the motor configuration
+Next, configure the left and right encoders as follows:
 
-Having configured a board and a motor component, you can now actuate your motor.
-Save the config by clicking **Save config** at the bottom of the page, then click over to the **Control** tab.
+{{< tabs name="Configure AMS-AS5048 Encoders" >}}
+{{% tab name="Config Builder" %}}
 
-Here you'll see a panel for the right `motor`.
-You'll use this panel to set the motor's `power` level.
+**Left encoder:**
```

Comment:
```suggestion
### Left encoder
```
Right now it almost looks like these are attributes or some such, since they're similar to the "Paste the following in the Attributes field:" line. Maybe use H4 if this is too big here?